core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -100,9 +100,15 @@ public int read(byte[] b) throws IOException {
 
   @Override
   public int read(byte[] b, int off, int len) throws IOException {
+    if (len == 0) {
```

Comment:
Add `Preconditions.check`  on `len >= 0` and `off >= 0`?

Reply:
Done.